src/Etg.Yams/Etg.Yams.nuspec

```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>Etg.Yams</id>
-    <version>1.8.0</version>
+    <version>1.9.0</version>
```

Comment:
1.8.0 has not been published yet, please don't bother with updating the version, I'll take care of it.